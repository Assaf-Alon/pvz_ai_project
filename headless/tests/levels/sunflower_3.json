{
    "10": [["normal", 3]],
    "20": [["normal", 3]],
    "30": [["normal", 3]],
    "60": [["normal", 3]]
}
