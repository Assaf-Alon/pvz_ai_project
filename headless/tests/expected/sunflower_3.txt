[0] Planted Sunflower in (3, 4). Total: 0.
[50] Sun generated by Sunflower in (3, 4). Total: 25.
[100] Sun generated by Sunflower in (3, 4). Total: 50.
[100] Normal zombie being spawned at lane 3
[100] Sun autospawned. Total: 75.
[101] Planted Sunflower in (3, 1). Total: 25.
[147] Normal zombie in (3, 9) moved to (3, 8).
[150] Sun generated by Sunflower in (3, 4). Total: 50.
[151] Sun generated by Sunflower in (3, 1). Total: 75.
[176] Planted Sunflower in (3, 0). Total: 25.
[194] Normal zombie in (3, 8) moved to (3, 7).
[200] Sun generated by Sunflower in (3, 4). Total: 50.
[200] Normal zombie being spawned at lane 3
[200] Sun autospawned. Total: 75.
[201] Sun generated by Sunflower in (3, 1). Total: 100.
[226] Sun generated by Sunflower in (3, 0). Total: 125.
[241] Normal zombie in (3, 7) moved to (3, 6).
[247] Normal zombie in (3, 9) moved to (3, 8).
[250] Sun generated by Sunflower in (3, 4). Total: 150.
[251] Sun generated by Sunflower in (3, 1). Total: 175.
[251] Planted Sunflower in (3, 2). Total: 125.
[276] Sun generated by Sunflower in (3, 0). Total: 150.
[288] Normal zombie in (3, 6) moved to (3, 5).
[294] Normal zombie in (3, 8) moved to (3, 7).
[300] Sun generated by Sunflower in (3, 4). Total: 175.
[300] Normal zombie being spawned at lane 3
[300] Sun autospawned. Total: 200.
[301] Sun generated by Sunflower in (3, 1). Total: 225.
[301] Sun generated by Sunflower in (3, 2). Total: 250.
[326] Sun generated by Sunflower in (3, 0). Total: 275.
[326] Planted Sunflower in (3, 5). Total: 225.
[327] Normal zombie in (3, 5) attacked.
[327] Sunflower in (3, 5) got damaged. HP: 200
[337] Normal zombie in (3, 5) attacked.
[337] Sunflower in (3, 5) got damaged. HP: 100
[341] Normal zombie in (3, 7) moved to (3, 6).
[347] Normal zombie in (3, 5) attacked.
[347] Sunflower in (3, 5) got damaged. HP: 0
[347] Sunflower in (3, 5) died.
[347] Normal zombie in (3, 9) moved to (3, 8).
[350] Sun generated by Sunflower in (3, 4). Total: 250.
[351] Sun generated by Sunflower in (3, 1). Total: 275.
[351] Sun generated by Sunflower in (3, 2). Total: 300.
[376] Sun generated by Sunflower in (3, 0). Total: 325.
[388] Normal zombie in (3, 6) moved to (3, 5).
[394] Normal zombie in (3, 5) moved to (3, 4).
[394] Normal zombie in (3, 8) moved to (3, 7).
[400] Sun generated by Sunflower in (3, 4). Total: 350.
[400] Sun autospawned. Total: 375.
[401] Sun generated by Sunflower in (3, 1). Total: 400.
[401] Sun generated by Sunflower in (3, 2). Total: 425.
[404] Normal zombie in (3, 4) attacked.
[404] Sunflower in (3, 4) got damaged. HP: 200
[414] Normal zombie in (3, 4) attacked.
[414] Sunflower in (3, 4) got damaged. HP: 100
[424] Normal zombie in (3, 4) attacked.
[424] Sunflower in (3, 4) got damaged. HP: 0
[424] Sunflower in (3, 4) died.
[425] Planted Sunflower in (3, 4). Total: 375.
[426] Sun generated by Sunflower in (3, 0). Total: 400.
[434] Normal zombie in (3, 4) attacked.
[434] Sunflower in (3, 4) got damaged. HP: 200
[435] Normal zombie in (3, 5) moved to (3, 4).
[441] Normal zombie in (3, 7) moved to (3, 6).
[444] Normal zombie in (3, 4) attacked.
[444] Sunflower in (3, 4) got damaged. HP: 100
[445] Normal zombie in (3, 4) attacked.
[445] Sunflower in (3, 4) got damaged. HP: 0
[445] Sunflower in (3, 4) died.
[451] Sun generated by Sunflower in (3, 1). Total: 425.
[451] Sun generated by Sunflower in (3, 2). Total: 450.
[476] Sun generated by Sunflower in (3, 0). Total: 475.
[488] Normal zombie in (3, 6) moved to (3, 5).
[491] Normal zombie in (3, 4) moved to (3, 3).
[492] Normal zombie in (3, 4) moved to (3, 3).
[500] Sun autospawned. Total: 500.
[500] Planted Sunflower in (3, 3). Total: 450.
[501] Sun generated by Sunflower in (3, 1). Total: 475.
[501] Sun generated by Sunflower in (3, 2). Total: 500.
[501] Normal zombie in (3, 3) attacked.
[501] Sunflower in (3, 3) got damaged. HP: 200
[502] Normal zombie in (3, 3) attacked.
[502] Sunflower in (3, 3) got damaged. HP: 100
[511] Normal zombie in (3, 3) attacked.
[511] Sunflower in (3, 3) got damaged. HP: 0
[511] Sunflower in (3, 3) died.
[526] Sun generated by Sunflower in (3, 0). Total: 525.
[535] Normal zombie in (3, 5) moved to (3, 4).
[549] Normal zombie in (3, 3) moved to (3, 2).
[551] Sun generated by Sunflower in (3, 1). Total: 550.
[551] Sun generated by Sunflower in (3, 2). Total: 575.
[558] Normal zombie in (3, 3) moved to (3, 2).
[559] Normal zombie in (3, 2) attacked.
[559] Sunflower in (3, 2) got damaged. HP: 200
[568] Normal zombie in (3, 2) attacked.
[568] Sunflower in (3, 2) got damaged. HP: 100
[569] Normal zombie in (3, 2) attacked.
[569] Sunflower in (3, 2) got damaged. HP: 0
[569] Sunflower in (3, 2) died.
[575] Planted Sunflower in (3, 2). Total: 525.
[576] Sun generated by Sunflower in (3, 0). Total: 550.
[578] Normal zombie in (3, 2) attacked.
[578] Sunflower in (3, 2) got damaged. HP: 200
[579] Normal zombie in (3, 2) attacked.
[579] Sunflower in (3, 2) got damaged. HP: 100
[582] Normal zombie in (3, 4) moved to (3, 3).
[588] Normal zombie in (3, 2) attacked.
[588] Sunflower in (3, 2) got damaged. HP: 0
[588] Sunflower in (3, 2) died.
[600] Normal zombie being spawned at lane 3
[600] Sun autospawned. Total: 575.
[601] Sun generated by Sunflower in (3, 1). Total: 600.
[626] Sun generated by Sunflower in (3, 0). Total: 625.
[626] Normal zombie in (3, 2) moved to (3, 1).
[629] Normal zombie in (3, 3) moved to (3, 2).
[635] Normal zombie in (3, 2) moved to (3, 1).
[636] Normal zombie in (3, 1) attacked.
[636] Sunflower in (3, 1) got damaged. HP: 200
[645] Normal zombie in (3, 1) attacked.
[645] Sunflower in (3, 1) got damaged. HP: 100
[646] Normal zombie in (3, 1) attacked.
[646] Sunflower in (3, 1) got damaged. HP: 0
[646] Sunflower in (3, 1) died.
[647] Normal zombie in (3, 9) moved to (3, 8).
[650] Planted Sunflower in (3, 1). Total: 575.
[655] Normal zombie in (3, 1) attacked.
[655] Sunflower in (3, 1) got damaged. HP: 200
[656] Normal zombie in (3, 1) attacked.
[656] Sunflower in (3, 1) got damaged. HP: 100
[665] Normal zombie in (3, 1) attacked.
[665] Sunflower in (3, 1) got damaged. HP: 0
[665] Sunflower in (3, 1) died.
[676] Sun generated by Sunflower in (3, 0). Total: 600.
[676] Normal zombie in (3, 2) moved to (3, 1).
[694] Normal zombie in (3, 8) moved to (3, 7).
[700] Sun autospawned. Total: 625.
[703] Normal zombie in (3, 1) moved to (3, 0).
[712] Normal zombie in (3, 1) moved to (3, 0).
[713] Normal zombie in (3, 0) attacked.
[713] Sunflower in (3, 0) got damaged. HP: 200
[722] Normal zombie in (3, 0) attacked.
[722] Sunflower in (3, 0) got damaged. HP: 100
[723] Normal zombie in (3, 0) attacked.
[723] Sunflower in (3, 0) got damaged. HP: 0
[723] Sunflower in (3, 0) died.
[723] Normal zombie in (3, 1) moved to (3, 0).
[725] Planted Sunflower in (3, 0). Total: 575.
[732] Normal zombie in (3, 0) attacked.
[732] Sunflower in (3, 0) got damaged. HP: 200
[733] Normal zombie in (3, 0) attacked.
[733] Sunflower in (3, 0) got damaged. HP: 100
[733] Normal zombie in (3, 0) attacked.
[733] Sunflower in (3, 0) got damaged. HP: 0
[733] Sunflower in (3, 0) died.
[741] Normal zombie in (3, 7) moved to (3, 6).
[779] Normal zombie in (3, 0) attempted entering the house.
[779] Normal zombie in (3, 0) triggered a lawnmower.
[779] Lawnmower in (3, 0) attacked.
[779] Normal zombie in (3, 0) was damaged. HP: -9818.
[779] Normal zombie in (3, 0) was killed.
[779] Normal zombie in (3, 0) was damaged. HP: -9818.
[779] Normal zombie in (3, 0) was killed.
[779] Normal zombie in (3, 0) was damaged. HP: -9818.
[779] Normal zombie in (3, 0) was killed.
[780] Lawnmower in (3, 0) moved to (3, 1).
[782] Lawnmower in (3, 1) moved to (3, 2).
[784] Lawnmower in (3, 2) moved to (3, 3).
[786] Lawnmower in (3, 3) moved to (3, 4).
[788] Lawnmower in (3, 4) moved to (3, 5).
[788] Normal zombie in (3, 6) moved to (3, 5).
[789] Lawnmower in (3, 5) attacked.
[789] Normal zombie in (3, 5) was damaged. HP: -9818.
[789] Normal zombie in (3, 5) was killed.
[789] You've won!
